sources/packages/backend/libs/sims-db/src/entities/disbursement-overaward.model.ts

```diff
@@ -97,4 +100,31 @@ export class DisbursementOveraward extends RecordDataModel {
     nullable: true,
   })
   deletedAt?: Date;
+  /**
+   * Note entered during manual overaward record added.
+   */
+  @OneToOne(() => Note, { eager: false, cascade: true })
```

Comment:
I believe that we had some back and forth along the time, but I believe that we should use `cascade: ["update"]` or even `cascade: false`.
@ann-aot @dheepak-aot thoughts?

Reply:
ya, I think we are following what you said